dist/typography.css

```diff
@@ -171,6 +171,11 @@ we are keeping these classes in code base to support legacy code
   color: var(--secondaryClr130);
 }
 
+/* bg-color classes */
+
+.bgClrLightGreen{
```

Comment:
why className is not primaryClr10 like other classes?

Reply:
Followed clrGreen's syntax @dheerajsingh123.

Reply:
okay